src/mots/cli.py

```diff
@@ -211,6 +258,8 @@ def create_parser():
         (main_cli, check_hashes, "check mots config and export hashes"),
         (main_cli, query, "query the module directory"),
         (main_cli, export, "export the module directory"),
+        (main_cli, set_default, "update settings variable and save to disk"),
+        (main_cli, get_default, "get settings variable, or all if no key is provided"),
```

Comment:
A comment on the design: maybe this should be a `mots settings` command with `get` and `set` subcommands - `mots settings` shows all settings after loading the config file, `mots settings get <setting>` shows a specific setting etc.

At minimum I think we should change the nomenclature to be referencing `settings` instead of `defaults`.

Reply:
Used `read` and `write` to keep implementation simple and avoid using `set` as the method name.